about/templates/about/index.html

```diff
@@ -54,17 +58,17 @@
 		</div>
 	</div>
 
-<script>
-if (document.width >= 768) {
-	$( document ).ready(function() {
-	    var heights = $(".member").map(function() {
-	        return $(this).height();
-	    }).get(),
-
-	    maxHeight = Math.max.apply(null, heights);
-
-	    $(".member").height(maxHeight);
-	});
-}
-</script>
+<!-- // <script>
```

Comment:
This script worked fine for me in Safari. We should be figuring out why it didn't work in other browsers then fixing it, instead of just scrapping it.